web/themes/custom/contribtracker/cypress/support/e2e.js

```diff
@@ -1 +1,29 @@
 import './commands';
+
+require('cypress-grep')();
+require('cypress-xpath');
+
+Cypress.on('uncaught:exception', () => {
+  // returning false here prevents Cypress from
+  // failing the test
+  return false;
+});
+
+import addContext from 'mochawesome/addContext';
+
+const titleToFileName = (title) => title.replace(/[:\/]/g, '');
+
+Cypress.on('test:after:run', (test, runnable) => {
+  if (test.state === 'failed') {
+    let parent = runnable.parent;
+    let filename = '';
+    while (parent && parent.title) {
+      filename = `${titleToFileName(parent.title)} -- ${filename}`;
+      parent = parent.parent;
+    }
+    filename += `${titleToFileName(test.title)} (failed).png`;
+    addContext({ test }, `../screenshots/${Cypress.spec.name}/${filename}`);
```

Comment:
Isn't this something Cypress does by default?

Reply:
Yes, it seems this is done to pass data to [mochawesome-report-generator](https://github.com/adamgruber/mochawesome-report-generator). I suggest not using any reporter for now. we can evaluate and consider it later.

Reply:
This is for the later use to provide an HTML report.

Reply:
Let's have a separate discussion and PR for reporting. In this PR, we should only focus on writing tests. We can address reporting later on.